images/ceph/Makefile

```
@@ -18,9 +18,9 @@ include ../image.mk
 # Image Build Options
 
 ifeq ($(GOARCH),amd64)
-CEPH_VERSION ?= v18.2.4-20240723
+CEPH_VERSION ?= v19.2.0-20240927
 else
-CEPH_VERSION ?= v18.2.4-20240724
+CEPH_VERSION ?= v19.2.0-20240927
```

```suggestion
CEPH_VERSION ?= v19.2.0-20240927
```

`v19.2.1-20250202` is available for x86 and arm